### ISSUE TITLE
[question]: The image lost many detail after upscale.

### ISSUE THREAD

**Issue by cavaniho** [vladmandic/automatic#2218]
### Issue Description

After I updated the latest version, the enlarged image details were relatively missing.

![00485-head out ou frame 1guy close up the](https://github.com/vladmandic/automatic/assets/107540026/2b1612eb-93d3-465f-9579-fd39da1117ec)
![02137-full shot white slime monster inside manly strong](https://github.com/vladmandic/automatic/assets/107540026/de30049b-9bbe-4092-be6e-1ba1ba6726a6)

The image above is before the update, and the image below is after the update.

### Version Platform Description

09:31:59-483001 INFO     Starting SD.Next
09:31:59-489345 INFO     Python 3.10.11 on Windows
09:31:59-597717 INFO     Version: app=sd.next updated=2023-09-17 hash=e0c8d37d
                         url=https://github.com/vladmandic/automatic.git/tree/master
09:32:00-261403 INFO     Latest published version: 792893e68e357088046b4e652919b491496e1ab0 2023-09-17T19:36:05Z
09:32:00-266783 INFO     Platform: arch=AMD64 cpu=Intel64 Family 6 Model 183 Stepping 1, GenuineIntel system=Windows
                         release=Windows-10-10.0.22621-SP0 python=3.10.11
09:32:00-269783 INFO     nVidia CUDA toolkit detected
09:32:00-840485 INFO     Extensions: disabled=[]
09:32:00-841483 INFO     Extensions: enabled=['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'stable-diffusion-webui-images-browser',
                         'stable-diffusion-webui-rembg', 'SwinIR'] extensions-builtin
09:32:00-847483 INFO     Extensions: enabled=[' (2).placeholder', 'a1111-sd-webui-tagcomplete', 'adetailer',
                         'Config-Presets', 'prompt-fusion-extension', 'sd-dynamic-prompts', 'sd-dynamic-thresholding',
                         'sd-webui-3d-open-pose-editor', 'sd-webui-additional-networks', 'sd-webui-neutral-prompt',
                         'sd-webui-prompt-all-in-one', 'sd-webui-supermerger', 'stable-diffusion-webui-anti-burn']
                         extensions
09:32:00-851483 INFO     Verifying requirements
09:32:00-867587 INFO     Verifying packages
09:32:00-869584 INFO     Verifying repositories
09:32:06-032992 INFO     Verifying submodules
09:32:36-397467 INFO     Extensions enabled: ['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'stable-diffusion-webui-images-browser',
                         'stable-diffusion-webui-rembg', 'SwinIR', ' (2).placeholder', 'a1111-sd-webui-tagcomplete',
                         'adetailer', 'Config-Presets', 'prompt-fusion-extension', 'sd-dynamic-prompts',
                         'sd-dynamic-thresholding', 'sd-webui-3d-open-pose-editor', 'sd-webui-additional-networks',
                         'sd-webui-neutral-prompt', 'sd-webui-prompt-all-in-one', 'sd-webui-supermerger',
                         'stable-diffusion-webui-anti-burn']
09:32:36-401478 INFO     Verifying packages
09:32:36-435806 INFO     Extension preload: {'extensions-builtin': 0.01, 'extensions': 0.0}
09:32:36-437805 INFO     Command line args: []
09:32:50-893424 INFO     Engine: backend=Backend.ORIGINAL compute=cuda mode=no_grad device=cuda
09:32:50-946784 INFO     Device: device=NVIDIA GeForce RTX 4080 Laptop GPU n=1 arch=compute_37 cap=(8, 9) cuda=11.8
                         cudnn=8700 driver=536.67
09:32:53-405657 INFO     Available VAEs: models\VAE items=12
09:32:53-444813 INFO     Available models: models\Stable-diffusion items=30 time=0.03s
09:32:57-284004 INFO     Extension: script='extensions-builtin\sd-webui-controlnet\scripts\controlnet.py' Warning:
                         ControlNet failed to load SGM - will use LDM instead.
09:32:57-286999 INFO     Extension: script='extensions-builtin\sd-webui-controlnet\scripts\controlnet.py' ControlNet
                         preprocessor location: E:\ai
                         drawing\automatic\extensions-builtin\sd-webui-controlnet\annotator\downloads
09:32:57-300999 INFO     Extension: script='extensions-builtin\sd-webui-controlnet\scripts\hook.py' Warning: ControlNet
                         failed to load SGM - will use LDM instead.
09:32:59-304418 INFO     Extension: script='extensions\a1111-sd-webui-tagcomplete\scripts\tag_autocomplete_helper.py'
                         Tag Autocomplete: Could not locate model-keyword extension, Lora trigger word completion will
                         be limited to those added through the extra networks menu.
09:33:02-597170 INFO     Extension: script='extensions\adetailer\scripts\!adetailer.py' [-] ADetailer initialized.
                         version: 23.9.1, num models: 9
100%|██████████████████████████████████████████████████████████████████████████████████| 6/6 [00:00<00:00, 3004.52it/s]
09:33:03-299202 INFO     Extension: script='extensions\sd-webui-additional-networks\scripts\additional_networks.py'
                         [AddNet] Updating model hashes...
100%|██████████████████████████████████████████████████████████████████████████████████| 6/6 [00:00<00:00, 3114.58it/s]
09:33:03-313630 INFO     Extension: script='extensions\sd-webui-additional-networks\scripts\model_util.py' [AddNet]
                         Updating model hashes...
09:33:04-473797 INFO     Extension: script='extensions\sd-webui-prompt-all-in-one\scripts\on_app_started.py'
                         sd-webui-prompt-all-in-one background API service started successfully.
09:33:05-938894 INFO     Extensions time: 12.32s { a1111-sd-webui-lycoris=1.64s clip-interrogator-ext=0.07s LDSR=0.05s
                         Lora=0.39s sd-extension-system-info=0.07s sd-webui-agent-scheduler=0.67s
                         sd-webui-controlnet=0.75s stable-diffusion-webui-images-browser=0.22s
                         stable-diffusion-webui-rembg=1.51s SwinIR=0.06s a1111-sd-webui-tagcomplete=0.20s
                         adetailer=3.29s prompt-fusion-extension=0.19s sd-dynamic-prompts=0.07s
                         sd-dynamic-thresholding=0.07s sd-webui-3d-open-pose-editor=0.11s
                         sd-webui-additional-networks=0.26s sd-webui-neutral-prompt=1.04s
                         sd-webui-prompt-all-in-one=0.12s sd-webui-supermerger=1.33s
                         stable-diffusion-webui-anti-burn=0.06s ScuNET=0.07s }
09:33:12-053527 INFO     Loading UI theme: name=step-3-profit/Midnight-Deep style=Auto
09:33:12-918720 INFO     Themes: builtin=6 default=5 external=41
09:33:33-416494 INFO     Local URL: http://127.0.0.1:7860/
09:33:33-419436 INFO     Initializing middleware
09:33:33-739923 INFO     [AgentScheduler] Task queue is empty
09:33:33-742917 INFO     [AgentScheduler] Registering APIs
09:33:35-103189 WARNING  Selected checkpoint not found: epicMixV3_v80.safetensors [81ce52db28]
Loading weights: E:\ai drawing\automatic\models\Stable-diffusion\ACertainty.ckpt ━╺━━━━━━━━━━━━━━━━━━ 0.2/4.3 GB 0:00:00
09:33:39-084547 INFO     Setting Torch parameters: device=cuda dtype=torch.float16 vae=torch.float16 unet=torch.float16
                         context=no_grad fp16=True bf16=False
09:33:39-831351 INFO     LDM: LatentDiffusion: Running in eps-prediction mode
09:33:39-834203 INFO     LDM: DiffusionWrapper has 859.52 M params.
Calculating model hash: E:\ai drawing\automatic\models\Stable-diffusion\ACertainty.ckpt ━━━━━━━━━━━━━ 4.3/4.3 GB 0:00:00
Loading weights: models\VAE\color101VAE_v1.pt ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╺━━━━━━━━ 261.6/334.7 MB 0:00:00
09:33:47-801230 INFO     Applying scaled dot product cross attention optimization
09:33:48-767941 INFO     Loaded embeddings: loaded=28 skipped=1
09:33:48-782002 INFO     Model loaded in 13.68s { load=3.85s config=0.16s create=0.73s apply=0.83s vae=5.91s move=1.21s
                         embeddings=0.97s }
09:33:49-259843 INFO     Model load finished: {'ram': {'used': 5.31, 'total': 31.74}, 'gpu': {'used': 3.26, 'total':
                         11.99}, 'retries': 0, 'oom': 0} cached=0
09:33:49-565676 INFO     Startup time: 73.09s { torch=9.01s gradio=2.10s diffusers=2.39s libraries=3.43s
                         extensions=12.32s codeformer=0.13s onchange=6.09s ui-txt2img=0.46s ui-img2img=0.23s
                         ui-settings=0.21s ui-extensions=2.65s ui-defaults=0.54s launch=17.19s api=0.10s
                         app-started=1.57s checkpoint=14.45s }
09:34:07-461770 INFO     Themes: builtin=6 default=5 external=41

### Relevant log output

```shell
2023-09-18T12:52:00.800Z	INFO	sd	devices	GPU high memory utilization: 100% {'ram': {'used': 8.34, 'total': 31.74}, 'gpu': {'used': 11.99, 'total': 11.99}, 'retries': 0, 'oom': 0}
2023-09-18T12:52:21.037Z	INFO	sd	devices	GPU high memory utilization: 100% {'ram': {'used': 8.32, 'total': 31.74}, 'gpu': {'used': 11.99, 'total': 11.99}, 'retries': 0, 'oom': 0}
2023-09-18T12:52:40.076Z	INFO	sd	devices	GPU high memory utilization: 100% {'ram': {'used': 8.4, 'total': 31.74}, 'gpu': {'used': 11.99, 'total': 11.99}, 'retries': 0, 'oom': 0}
2023-09-18T12:52:40.748Z	INFO	sd	processing	Processed: images=3 time=60.85s its=1.48 memory={'ram': {'used': 3.4, 'total': 31.74}, 'gpu': {'used': 3.79, 'total': 11.99}, 'retries': 0, 'oom': 0}
```

### Backend

Original

### Model

SD 1.5

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by vladmandic** [comment]
what is "after being restored to upscale"? please write exact steps you've performed. 
and your "relevant log output" does not show upscaling operation at all.

**Comment by cavaniho** [comment]
> what is "after being restored to upscale"? please write exact steps you've performed. and your "relevant log output" does not show upscaling operation at all.
> 
> and looking at metadata, images are using different upscalers: `4x_NMKD-Superscale-SP_178000_G` vs `ToonVHS-1x-300000_G`. if we're comparing, lets compare same things.

My English skills are not that great, so I currently rely on translation software. As a result, there might be some weird expressions, but I'll try my best to explain the problem.
First, I entered the prompt and selected the options of advance and second pass. Additionally, I utilized upscaling before generating. But, the images it creates aren't that great in terms of quality. I have used different upscalers and got the same results.
I reckon, based on what you said, that it's possible the upscaling operation didn't run in the background after the version update.
